src/wp-includes/class-wp-scripts.php

```diff
@@ -1028,36 +1076,54 @@ private function has_only_delayed_dependents( $handle, $async_only = false, $che
 		$checked[ $handle ] = true;
 		$dependents         = $this->get_dependents( $handle );
 
-		// If there are no dependents remaining to consider, the script can be deferred.
+		// If there are no dependents remaining to consider, the script can be delayed.
 		if ( empty( $dependents ) ) {
 			return true;
 		}
 
 		// Consider each dependent and check if it is delayed.
 		foreach ( $dependents as $dependent ) {
-			// If the dependent script has no src (as it represents a script bundle), ignore it for consideration.
-			if ( empty( $this->registered[ $dependent ]->src ) ) {
+			if ( ! array_key_exists( $dependent, $this->registered ) ) {
```

Comment:
https://github.com/WordPress/wordpress-develop/pull/4391/files?file-filters%5B%5D=.php&show-viewed-files=true#r1232052943

Reply:
Done in 47b2cbccfc7641d141994c5ec0a28e356d12da3e